src/core/tests/type_prop/strided_slice.cpp

```diff
@@ -496,7 +496,13 @@ TEST_P(StridedSliceShapeInferTest, begin_end_strides_are_not_constants) {
     const auto input_data = std::make_shared<op::v0::Parameter>(params.ref_type, params.input_shape);
     const auto begin = std::make_shared<op::v0::Parameter>(ov::element::i32, params.begin_shape);
     const auto end = std::make_shared<op::v0::Parameter>(ov::element::i32, params.end_shape);
-    const auto strides = std::make_shared<op::v0::Parameter>(ov::element::i32, params.strides_shape);
+    const auto strides = params.strides_shape
+                             ? std::static_pointer_cast<Node>(
+                                   std::make_shared<op::v0::Parameter>(ov::element::i32, *params.strides_shape))
+                             : std::static_pointer_cast<Node>(
+                                   ov::op::v0::Constant::create(ov::element::i32,
+                                                                Shape{params.begin_mask.size()},
+                                                                std::vector<int32_t>(params.begin_mask.size(), 1)));
```

Comment:
The test case name is "...strides_are_not_constants" while this modification makes the tested strides Constant, so to avoid further confusion it would be better to create a separate test case for it or adjust the test name.
Worth to test a `stride != 1` as well.

https://github.com/openvinotoolkit/openvino/blob/66f79c0b39a45afb006725b828b0355bd65a7b50/src/core/tests/type_prop/strided_slice.cpp#L493